### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Algebra/Subalgebra/Basic.lean [hunk 1]
```diff
@@ -626,14 +626,12 @@ def rangeRestrict (f : A →ₐ[R] B) : A →ₐ[R] f.range :=
   f.codRestrict f.range f.mem_range_self
 #align alg_hom.range_restrict AlgHom.rangeRestrict
 
-theorem rangeRestrict_surjective (f : A →ₐ[R] B):
-    Function.Surjective (f.rangeRestrict) :=
-  fun ⟨_y, hy⟩ =>
+theorem rangeRestrict_surjective (f : A →ₐ[R] B) :
+    Function.Surjective (f.rangeRestrict) := fun ⟨_y, hy⟩ =>
```
Comment (sgouezel) [inline Mathlib/Algebra/Algebra/Subalgebra/Basic.lean:630]: I'm not convinced by this one: I'd rather have the proof cleanly separated from the statement, unless it fits on one single line.

File: Mathlib/Topology/Baire/CompleteMetrizable.lean [hunk 2]
```diff
@@ -71,15 +71,15 @@ instance (priority := 100) BaireSpace.of_pseudoEMetricSpace_completeSpace : Bair
   let r : ℕ → ℝ≥0∞ := fun n => (F n).2
   have rpos : ∀ n, 0 < r n := by
     intro n
-    induction' n with n hn
-    exact lt_min εpos (Bpos 0)
-    exact Hpos n (c n) (r n) hn.ne'
+    induction n with
```
Comment (sgouezel) [inline Mathlib/Topology/Baire/CompleteMetrizable.lean:74]: do you really need `induction`, or is `cases` enough here?

**Review by sgouezel — COMMENTED** [review]
bors d+
Thanks!

File: Mathlib/Topology/Baire/CompleteMetrizable.lean (hunk 2 above)
Comment (Ruben-VandeVelde) [reply Mathlib/Topology/Baire/CompleteMetrizable.lean:74]: We really do, yeah.

**Comment by Ruben-VandeVelde** [comment]
bors merge

**Pull request closed without merging**